scholly/service/core/push.js

```diff
@@ -68,22 +68,33 @@ async function makeGHRepo(ghUsername, ghToken, repoName) {
     Authorization: `token ${ghToken}`,
     Accept: 'application/vnd.github.v3+json',
   };
-  let repoResponse;
+
+  let repoExists;
   try {
-    repoResponse = await axios.get(
+    const repoResponse = await axios.get(
       `https://api.github.com/search/repositories?${searchQuery}`,
       {
         headers: options,
       },
     );
+    repoExists = repoResponse.data.total_count === 1;
   } catch (err) {
-    console.log(err);
+    if (err.response.status !== 422) {
+      // Error 422 is fine (it can be caused by user having no repo or only private ones in their account), any other should be thrown back up.
+      winston.error(
+        `Error checking existing repositories for user: ${ghUsername}`,
+        err,
```

Comment:
I believe passing `err` as an arg to the logger doesn't do what you're trying to do here. We might have to interpolate the `err.message` as part of the initial logging message.